scripts/components/networking/statssynchronizercomponent/StatsSynchronizerComponent.gd

```diff
@@ -426,6 +438,50 @@ func remove_boost(boost: Boost):
 	for watcher in watcher_synchronizer.watchers:
 		G.sync_rpc.statssynchronizer_sync_response.rpc_id(watcher.peer_id, target_node.name, data)
 
+	boosts_changed.emit()
+
+
+func reload_boosts():
+	load_defaults()
+
+	var flatValues: Dictionary = {}
```

Comment:
try to match the style of the rest of the file. This file uses snake-case namings instead of camel-case.

Reply:
I usually make exclusively "private" variables camelCase since nothing else can access them (so if anyone tried, it would instantly look off), but i can change them to snake_case

Reply:
I understand it doesn't really impact anything functionally but when working on larger projects (and this one is getting big) with multiple people it's best to "enforce" naming conventions to make the code easier to read/consistent.

I follow this GDScript style guide:
https://docs.godotengine.org/en/stable/tutorials/scripting/gdscript/gdscript_styleguide.html

Which states that variables should be snake_case.